pandas/io/pytables.py

```diff
@@ -580,16 +580,37 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         self.close()
 
-    def keys(self) -> List[str]:
+    def keys(self, include: Optional[str] = "pandas") -> List[str]:
```

Comment:
this is not Optional, just str (alternatively, we could actually do ```Optiona[str] = None``` and then allow passing include='native')

but find either way

Reply:
Removed the `Optional` annotation